checkov/terraform/runner.py

```diff
@@ -292,7 +290,7 @@ def get_graph_checks_report(self, root_folder: str, runner_filter: RunnerFilter)
                     report.add_record(record=record)
         return report
 
-    def get_entity_context_and_evaluations(self, entity):
+    def get_entity_context_and_evaluations(self, entity: dict[str, Any]) -> tuple[dict[str, Any] | None, None]:
```

Comment:
the second return type is always `None`? seems we can just drop it no?

Reply:
yeah, it looks like. Checked current usage, also in downstream projects, and it seems not to be set in a subclass.